doc/SPO-MANUAL-MIGRATION-GUIDE.md

```diff
@@ -0,0 +1,564 @@
+# Backup seccomp profiles
+for profile in $(oc get seccompprofiles -n openshift-security-profiles -o name); do
+  name=$(echo $profile | cut -d'/' -f2)
+  oc get seccompprofile $name -n openshift-security-profiles -o json | jq '.' > \
+    "$BACKUP_DIR/profiles/seccompprofiles-openshift-security-profiles-${name}.json"
+  echo "Backed up seccompprofile: $name"
+done
+
+# Backup SELinux profiles
+for profile in $(oc get selinuxprofiles -n openshift-security-profiles -o name); do
+  name=$(echo $profile | cut -d'/' -f2)
+  oc get selinuxprofile $name -n openshift-security-profiles -o json | jq '.' > \
+    "$BACKUP_DIR/profiles/selinuxprofiles-openshift-security-profiles-${name}.json"
+  echo "Backed up selinuxprofile: $name"
+done
+
+# Backup raw SELinux profiles
+for profile in $(oc get rawselinuxprofiles -n openshift-security-profiles -o name); do
+  name=$(echo $profile | cut -d'/' -f2)
+  oc get rawselinuxprofile $name -n openshift-security-profiles -o json | jq '.' > \
+    "$BACKUP_DIR/profiles/rawselinuxprofiles-openshift-security-profiles-${name}.json"
+  echo "Backed up rawselinuxprofile: $name"
+done
+```
+
+**Actual output:**
+```
+Backed up seccompprofile: profile-block-all
+Backed up selinuxprofile: errorlogger-selinuxd-test
+Backed up rawselinuxprofile: errorlogger
+```
+
+### Create Backup Summary
+```bash
+echo "=== Backup Summary ===" > $BACKUP_DIR/backup-summary.txt
+echo "Total files backed up: $(find $BACKUP_DIR -name "*.json" | wc -l)" >> $BACKUP_DIR/backup-summary.txt
+echo "" >> $BACKUP_DIR/backup-summary.txt
+echo "SPOD configuration: $([ -f $BACKUP_DIR/spod-configuration.json ] && echo "Yes" || echo "No")" >> $BACKUP_DIR/backup-summary.txt
+echo "" >> $BACKUP_DIR/backup-summary.txt
+echo "Profiles backed up:" >> $BACKUP_DIR/backup-summary.txt
+find $BACKUP_DIR/profiles -name "*.json" -exec basename {} \; | sort >> $BACKUP_DIR/backup-summary.txt
+echo "" >> $BACKUP_DIR/backup-summary.txt
+echo "Profile bindings backed up:" >> $BACKUP_DIR/backup-summary.txt
+find $BACKUP_DIR/bindings-recordings -name "*.json" -exec basename {} \; | sort >> $BACKUP_DIR/backup-summary.txt
+
+cat $BACKUP_DIR/backup-summary.txt
+```
+
+**Actual output:**
+```
+=== Backup Summary ===
+Total files backed up: 6
+
+SPOD configuration: Yes
+
+Profiles backed up:
+rawselinuxprofiles-openshift-security-profiles-errorlogger.json
+seccompprofiles-openshift-security-profiles-profile-block-all.json
+selinuxprofiles-openshift-security-profiles-errorlogger-selinuxd-test.json
+
+Profile bindings backed up:
+profilebindings-openshift-security-profiles-profile-binding-seccomp.json
+profilebindings-openshift-security-profiles-profile-binding-selinuxd.json
+```
+
+## Step 4: Delete SPO Objects in Order
+
+**IMPORTANT**: Delete profile bindings first to prevent the operator from automatically reapplying profiles to workloads.
+
+### Delete Profile Bindings First
+```bash
+# List and delete all profile bindings
+oc get profilebindings -n openshift-security-profiles
```

Comment:
One more issue: before delete any profilebinding, the activeWorkloads should be deleted first. Otherwise, the profilebinding deleting will fail.